dwl.c

```diff
@@ -981,11 +981,50 @@ createpointer(struct wlr_input_device *device)
 		struct libinput_device *libinput_device =  (struct libinput_device*)
 			wlr_libinput_get_device_handle(device);
 
-		if (tap_to_click && libinput_device_config_tap_get_finger_count(libinput_device))
-			libinput_device_config_tap_set_enabled(libinput_device, LIBINPUT_CONFIG_TAP_ENABLED);
+		if (libinput_device_config_tap_get_finger_count(libinput_device)) {
+			if (tap_to_click)
+				libinput_device_config_tap_set_enabled(libinput_device, LIBINPUT_CONFIG_TAP_ENABLED);
+
+			if (tap_and_drag)
+				libinput_device_config_tap_set_drag_enabled(libinput_device, LIBINPUT_CONFIG_DRAG_ENABLED);
+			else
+				libinput_device_config_tap_set_drag_enabled(libinput_device, LIBINPUT_CONFIG_DRAG_DISABLED);
```

Comment:
I think is better just `libinput_device_config_tap_set_drag_enabled(libinput_device, tap_and_drag);`

Reply:
I wasn't sure if `LIBINPUT_CONFIG_DRAG_ENABLED` was guaranteed to be 1, etc. But if that's the case, you're right, this is much cleaner.  Perhaps the if statement for `tap_to_click` should be replaced with a similar line if `LIBINPUT_CONFIG_TAP_ENABLED` is always going to be 1?